examples/lib/intermediateHtmlStepUi.js

```diff
@@ -128,46 +116,42 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
           '<option value="' + m + '">' + modulesInfo[m].name + '</option>'
         );
     }
-    insertStepSelect.selectize({
-      sortField: 'text'
-    });
-    //$step('.inserDiv .add-step-btn').prop('disabled', true);
-    $(step.ui.querySelector('.inserDiv .add-step-btn')).prop('disabled', true);
+
+    $('.inserDiv .add-step-btn').prop('disabled', true);
     
     insertStepSelect.append('<option value="" disabled selected>Select a Module</option>');
     //$step('.insertDiv .radio-group .radio').on('click', function () {
```

Comment:
Can we get rid of this one too? `//$step('.insertDiv .radio-group .radio').on('click', function () {`

Reply:
Ya thanks will do that.